ArrayOps.java

```diff
@@ -1,26 +1,101 @@
 public class ArrayOps {
     public static void main(String[] args) {
-        
+    // System.out.println(findMissingInt(new int[] {1, 2}));
+    System.out.println(secondMaxValue(new int[] {1, -2, 3, -4, 5}));
+    // containsTheSameElements(new int[] {1, 5}, new int[] {5, 1, 5, 1, 5});
+    // isSorted(new int[] {1, -2, 3});
     }
     
+    
     public static int findMissingInt (int [] array) {
-        // Write your code here:
-        return -1;
+        int missingInt = -1;
+        boolean isMissing = false;
+        for (int i = 0; i < array.length + 1; i++) {    
+            for (int arrayIndex = 0; arrayIndex < array.length; arrayIndex++) {
```

Comment:
Better to define inner variables only in their scope.
I changed the initialize value to be true so you can remove the first condition - follow the next comment
```suggestion
        for (int i = 0; i < array.length + 1; i++) {    
            boolean isMissing = true;
            for (int arrayIndex = 0; arrayIndex < array.length; arrayIndex++) {
```